main.go

```diff
@@ -4,20 +4,25 @@ import (
 	"context"
 	"crypto/tls"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"mime"
 	"net"
 	"net/http"
 	"os"
 	"strings"
+	"time"
+
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
+
```

Comment:
Imports grouping seems a bit random there. golangci-lint would catch that.

Reply:
golangci-lint is not reporting anything wrong about imports
P.S goimports too

Reply:
Too bad, but grouping is still random. Why is that import not in the group below, for example?